runtime/onert/core/src/compiler/Fp32ToFp16Converter.cc

```diff
@@ -895,7 +921,9 @@ void Fp32ToFp16Converter::deleteContiguousOpSequences(
     VERBOSE(Fp32ToFp16Converter) << "Delete Node #" << first_node_ind.value() << std::endl;
 
     // Uses
-    for (auto &ind : first_node.getInputs())
+    std::unordered_set<ir::OperandIndex> unique_inputs(first_node.getInputs().begin(),
```

Comment:
Every time we have to make unordered_set for `for-each` loop? Supporting `for-each` loop easily looks better. Or any reason why we should not?

Reply:
Using `unordered_set` may be unified to a method later.